### PR TITLE
Corrected KeyPairName resource Type

### PR DESCRIPTION
**Pull request by ddomingorh**
https://bulldog.internal.atlassian.com/browse/DCD-965

### CODE REVIEW

File: templates/quickstart-bitbucket-dc-with-vpc.template.yaml [hunk 1]
```diff
@@ -526,8 +526,8 @@ Parameters:
     Type: String
   KeyPairName:
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
-    Description: The EC2 Key Pair to allow SSH access to the instances. 
-    Type: List<AWS::EC2::Keypair>
+    Description: The EC2 Key Pair to allow SSH access to the instances.
+    Type: List<AWS::EC2::Keypair::KeyName>
```
Comment (errcode1202) [inline templates/quickstart-bitbucket-dc-with-vpc.template.yaml]: `AWS::ec2::KeyPair::KeyName` returns an object of all available KeyPairs so no need to wrap in a List. Should be defined like so:

`Type: AWS::ec2::KeyPair::KeyName`

**Review by errcode1202 — CHANGES_REQUESTED** [review]
List definition not necessary

File: templates/quickstart-bitbucket-dc-with-vpc.template.yaml (hunk 1 above)
Comment (ddomingorh) [reply templates/quickstart-bitbucket-dc-with-vpc.template.yaml]: Done!

**Review by errcode1202 — APPROVED** [review]
🚀

**Pull request merged**